packages/nimble-components/src/table/components/cell/styles.ts

```diff
@@ -24,8 +24,6 @@ export const styles = css`
 
     .cell-view {
         overflow: hidden;
-        display: flex;
-        align-items: center;
```

Comment:
I moved the display configuration to the cell-view implementations. They should be responsible for their self and content styling